pandas/tests/frame/test_analytics.py

```diff
@@ -25,22 +25,44 @@
 import pandas.util._test_decorators as td
 
 
-def _check_stat_op(name, alternative, main_frame, float_frame,
-                   float_string_frame, has_skipna=True,
-                   has_numeric_only=False, check_dtype=True,
-                   check_dates=False, check_less_precise=False,
-                   skipna_alternative=None):
-
-    f = getattr(main_frame, name)
+def assert_stat_op_calc(opname, alternative, frame, has_skipna=True,
```

Comment:
* Just curious, why the name change?
* In general, I believe we keep the underscore unless we intend to export this function (which we aren't?)

Reply:
@gfyoung 
It came out of review from @jreback https://github.com/pandas-dev/pandas/pull/22733#discussion_r219699368: 
> can this just be a module level function now? maybe rename to assert_stat_ops

I then split it into `_calc` and `_api`. Happy to add an underscore or rename it to whatever.

Reply:
Gotcha.  My personal preference is underscore for the name given the reasoning I described above.  The changes are good otherwise.